cs.html

```diff
@@ -961,689 +1180,684 @@ <h2>Constructors</h2>
+              </dd>
+            </dl>
+          </section>
+        </div>
+      </section>
+    </div>
+  </section>
+
+  <section id="incoming-stream*">
+    <h2>Interface Mixin <dfn>IncomingStream</dfn></h2>
+    <p>
+        An OutgoingStream is a stream that can be read fromn, 
+        as either a <code>ReceiveStream </code>or a <code>BidirectionalStream</code>
+    </p>    
+    <div>
+      <pre class="idl">
+[ Exposed=Window ]
+interface mixin IncomingStream {
+    readonly attribute boolean readable;
+    readonly attribute unsigned long readableAmount;
+    readonly attribute Promise&lt;StreamAbortInfo&gt; readingAborted;
+    StreamReadResult readInto (Uint8Array data);
+    void abortReading (StreamAbortInfo abortInfo);
+    Promise&lt;void&gt;   waitForReadable(unsigned long amount);
+};</pre>
+      <section>
+        <h3>Overview</h3>
+        <p>The <code><a>IncomingStream</a></code> will initialize with
+        the following:</p>
+        <ol>
+          <li>Let <var>stream</var> be the <code><a>IncomingStream</a></code>.</li>
+          <li>Let <var>stream</var> have a <dfn>[[\Readable]]</dfn> internal
+          slot initialized to <code>true</code>.</li>
+          <li>Let <var>stream</var> have a <dfn>[[\ReadableAmount]]</dfn> internal
+          slot initialized to zero.</li>
+        </ol>
+      </section>
+      <section>
+        <h3>Attributes</h3>
+        <dl data-link-for="IncomingStream" data-dfn-for="IncomingStream" class=
+        "attributes">
+          <dt><code>readable</code> of type <span class="idlAttrType"><a>boolean</a></span>,
+          readonly</dt>
+          <dd>
+            <p>The <dfn id="dom-incomingstream-readableamount"><code>readable</code></dfn>
+            attribute represents whether data can be read from the <code><a>IncomingStream</a></code>.
+            On getting, it <em class="rfc2119" title="MUST">MUST</em> return the value of the
+            <code><a>IncomingStream</a></code>'s <a>[[\Readable]]</a> slot.
+          </dd>
+          <dt><code>readableAmount</code> of type <span class="idlAttrType"><a>unsigned
+          long</a></span>, readonly</dt>
+          <dd>
+            <p>The <dfn id="dom-incomingstream-readableamount"><code>readableAmount</code></dfn>
+            attribute represents the number of bytes buffered for access by
+            <code>readInto</code> but that, as of the last time the event loop
+            started executing a task, had not yet been read. This does not include
+            framing overhead incurred by the protocol, or buffers associated with
+            the network hardware. On getting, it <em class="rfc2119" title="MUST">MUST</em>
+            return the value of the <code><a>IncomingStream</a></code>'s
+            <a>[[\ReadableAmount]]</a> internal slot.</p>
+          </dd>
+          <dt><code>readingAborted</code> of type <span class="idlAttriType"><a>StreamAbortInfo</a>
```

Comment:
idlAttrType?  Also I think the span tag should be closed here.

Reply:
Good find.  Fixed.